stable/consul/templates/consul.yaml

```diff
@@ -119,9 +119,7 @@ spec:
           protocol: "UDP"
           containerPort: {{.Values.ConsulDnsPort}}
         resources:
-          requests:
-            cpu: "{{ .Values.Cpu }}"
-            memory: "{{ .Values.Memory }}"
+{{ toYaml .Values.resources | indent 10 }}
```

Comment:
Make it start with a upper case letter for consistency (even though that does wouldn't match our best practices).

`{{ toYaml .Values.Resources | indent 10 }}`